src/main/java/programminglife/model/Node.java

```diff
@@ -0,0 +1,70 @@
+package programminglife.model;
+
+import java.util.HashSet;
+import java.util.Scanner;
+import java.util.Set;
+
+/**
+ * Created by marti_000 on 25-4-2017.
+ */
+public class Node {
+    private int id;
+    private String sequence;
+
+    private Set<Node> parents;
+    private Set<Node> children;
+
+    public Node(int id) {
+        this(id, new String(), new HashSet<>(), new HashSet<>());
+    }
+
+    public Node(int id, String sequence) {
+        this(id, sequence, new HashSet<>(), new HashSet<>());
+    }
+
+    public Node(int id, String sequence, Set<Node> parents, Set<Node> children) {
+        this.id = id;
+        this.sequence = sequence;
+        this.parents = parents;
+        this.children = children;
+    }
+
+    public String getSequence() {
+        return sequence;
+    }
+
+    public void setSequence(String sequence) {
+        this.sequence = sequence;
+    }
+
+    public static Node parseSegment(Scanner sc) {
+        int id = Integer.parseInt(sc.next());
+        String segment = sc.next();
+        sc.next();
```

Comment:
Should be written in the javadoc. Checkstyle will gives warnings for this in the future